src/store/modules/order/actions.ts

```diff
@@ -0,0 +1,47 @@
+import { ActionTree } from 'vuex'
+import store from '@/store'
+import RootState from '@/store/RootState'
+import OrderState from './OrderState'
+import * as types from './mutation-types'
+import router from '@/router'
+
+
+const actions: ActionTree<OrderState, RootState> = {
+  updateOrderList ({ commit }, payload) {
+    commit(types.ORDER_LIST_UPDATED,  payload );
+  },
+  modifyCsv ({ commit }, payload) {
```

Comment:
We will be performing updates on order item list only. The state will not update any CSV, but order list

Reply:
Updated the code.